### ISSUE TITLE
No info in wiki about custom enchantments

### ISSUE THREAD

**Issue by MechanosG** [AzureDoom/AzureDoom-Doom-Fabric#12]
What does the Doom Leaping enchantment do?  I found some armor with it.  Looking at the github code, I'm not certain but it kind of looks like..  it's supposed to enable double jumping or something?  But double jumping seems to be already enabled for everyone?  :confounded: 

Also, putting Daisy in a Dank Storage caused a server to crash.  I _assume_ this is Dank Storage's problem and I'll probably be making an issue about it over there.  Just mentioning it since I'm already here.

Minecraft 1.16.4 Fabric
MCDoom 3.9.0
Dank Storage 1.9a

**Comment by AzureDoom** [comment]
Moved this to my Fabric repo since it's about the Fabric version. I will see what's up with the double jump, it's suppose to only be for the enchantment.

I will check Dank but I don't see why it would crash Dank tbh.

**Comment by MechanosG** [comment]
Oops, didn't realize I was in non-fabric repo.  Thanks for quick response.  There is a small chance the Daisy/Dank thing was coincidence, as I didn't have access to the server's crash log.  But it happened exactly when I tried to put a 2nd Daisy I had just found, into a red Dank.  When server came back up, my character lost about 60 seconds and the Daisy was no longer in the Dank. I didn't put it back in just in case.  I'll test in singleplayer and in a temp local server to see if I can reproduce it.
